packages/addons/tools/system-tools/package.mk

```diff
@@ -118,6 +119,10 @@ addon() {
     cp -P $(get_install_dir jq)/usr/bin/jq ${ADDON_BUILD}/${PKG_ADDON_ID}/bin
     cp -P $(get_install_dir oniguruma)/usr/lib/{libonig.so,libonig.so.5,libonig.so.5.2.0} ${ADDON_BUILD}/${PKG_ADDON_ID}/lib
 
+    # libgpiod
+    cp -P $(get_install_dir libgpiod)/usr/bin/{gpiodetect,gpiofind,gpioget,gpioinfo,gpiomon,gpioset} ${ADDON_BUILD}/${PKG_ADDON_ID}/bin
+    cp -L $(get_install_dir libgpiod)/usr/lib/libgpiod.so.2 ${ADDON_BUILD}/${PKG_ADDON_ID}/lib
```

Comment:
Could the tools just be built statically or do you think other tools will need the shared libgpiod library?

Reply:
sure, libgpiod can be linked statically. I don't think there many tools using the library.